src/include/hip/rpp_hip_common.hpp

```diff
@@ -1551,6 +1551,20 @@ __device__ __forceinline__ void rpp_hip_load24_pkd3_to_int24_pln3(schar *srcPtr,
 
 // /******************** DEVICE MATH HELPER FUNCTIONS ********************/
 
+// float8 min
+
+__device__ __forceinline__ void rpp_hip_math_min8(d_float8 *srcPtr_f8, float *dstPtr)
+{
+    *dstPtr = fminf(fminf(fminf(fminf(fminf(fminf(fminf(srcPtr_f8->f1[0], srcPtr_f8->f1[1]), srcPtr_f8->f1[2]), srcPtr_f8->f1[3]), srcPtr_f8->f1[4]), srcPtr_f8->f1[5]), srcPtr_f8->f1[6]), srcPtr_f8->f1[7]);
```

Comment:
please use vector fmin and fmax instead of nested

Reply:
@rrawther There is no fmin and fmax for 4 or 8 vector length in hip yet Rajy.
Just checking that correct still @AryanSalmanpour?